### PR TITLE
Tourism place

### PR DESCRIPTION
**Pull request by Ashok-mangali**
Please have a look my code

### CODE REVIEW

File: Favorite_Places/favorite_place.html [hunk 1]
```diff
@@ -1,25 +1,12 @@
 <html>
-<<<<<<< HEAD
```
Comment (kishorekumar646) [inline Favorite_Places/favorite_place.html]: Plaese remove conflicts code

File: README.md [hunk 2]
```diff
@@ -1,4 +1,22 @@
 # Adding Front End Projects
 > You can create pull request
+please 
+
+
```
Comment (kishorekumar646) [inline README.md:5]: Remove spaces

**Review by kishorekumar646 — CHANGES_REQUESTED** [review]
PR note

**Review by kishorekumar646 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Ashok-mangali) [reply README.md:5]: It's done

File: Favorite_Places/favorite_place.html (hunk 1 above)
Comment (Ashok-mangali) [reply Favorite_Places/favorite_place.html]: resolved conflicts

**Review by kishorekumar646 — APPROVED** [review]

**Review by kishorekumar646 — APPROVED** [review]

**Pull request merged**